src/Pages/Calculator.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import CalculatorComponent from '../component/CalculatorComponent';
+import NavBar from '../component/Navbar';
+
+const Calculator = () => (
+  <div>
+    <NavBar />
```

Comment:
- [ ] kindly remove the `navbar` in this component and import it in the app.js to call it for once instead of calling it in each component